### ISSUE TITLE
RSS Gadget not pulling CDATA tags.

### ISSUE THREAD

**Issue by ashleighalmeida** [Rise-Vision/gadgets#13]
When user tries to load a RSS URL using CDATA Tags, the presentation will only display headers without descriptions.

URL used: http://www.parishepiscopal.org/rss.cfm?news=53

![screen shot 2015-12-04 at 11 56 40 am](https://cloud.githubusercontent.com/assets/11686685/11595438/9bf7ced0-9a7d-11e5-9844-9e5720aff5be.png)

Presentation: http://rva.risevision.com/#/PRESENTATION_MANAGE/id=a6bc9ea0-1fb2-4260-ae53-a4b76d39395e?cid=39edb67e-21a0-47b5-9048-860887dc6a08

User should be able to display RSS Feeds that use CDATA Tags in order for the RSS Feed to display properly.

**Comment by lapinnoir** [comment]
Another one with CDATA in the RSS but the request is different - user wants to ensure that the titles are appearing as is but some problems displaying data

Original pres: http://rva.risevision.com/#/PRESENTATION_MANAGE/id=49195716-de8d-45bf-b8d6-258f9b295355?cid=3dc328b4-53a3-4b22-861d-130e3ad92e15

**Comment by ashleighalmeida** [comment]
https://app.intercom.io/a/apps/d85f4ah4/inbox/apa@risevision.com/conversations/1489329101

**Comment by lapinnoir** [comment]
https://app.intercom.io/a/apps/d85f4ah4/inbox/hcs@risevision.com/conversations/1369336864 (Intercome integration)

**Comment by ashleighalmeida** [comment]
https://app.intercom.io/a/apps/d85f4ah4/inbox/hcs@risevision.com/conversations/1369336864 (intercom)

**Comment by lapinnoir** [comment]
https://app.intercom.io/a/apps/d85f4ah4/inbox/hcs@risevision.com/conversations/1369336864

**Comment by donnapep** [comment]
@lapinnoir Can you clarify what "some problems displaying data" means? I am seeing green titles and blue descriptions so wondering if I'm missing something. Thx.

**Comment by donnapep** [comment]
Problem is in `RiseVision.Common.Utility.getNodeValue`. `CDATA` sometimes has several child nodes, and that function will return the first one, which is blank. I suspect that having blank lines and / or multiple lines within the CDATA causes this.

**Comment by lapinnoir** [comment]
@donnapep I've noticed in the RSS source multiple CDATA lines.  
```
<description><![CDATA[
							12/7/2015
							
								 3:30 PM
							
							
								 - 4:30 PM
							
							
							
						]]></description>
```

In http://rva.risevision.com/#/PRESENTATION_MANAGE/id=44b5df36-7112-4992-a803-9041a44dc242?cid=960b72df-9af7-476b-8a47-b67ded4c779d  I've posted a screenshot of the blank data

<img width="411" alt="rss screenshot 2015-12-04 at 5 20 09 pm" src="https://cloud.githubusercontent.com/assets/5868781/11602857/a58d090a-9aab-11e5-8fde-fd26e12f1b1a.png">

This is RSS gadget with no transitions set.

The current workaround I'm presenting to the user is either:
1. RSS gadget with transitions (but seems to have a 10 sec gap before starting again)
2. RSS gadget with transitions but have two in one Placeholder (this resolves the 10sec issue above but transitions seem to vary a bit even though they are the same so it's a bit erratic)
